Framework/src/ObjectsManager.cxx

```diff
@@ -31,22 +31,19 @@ namespace core
 ObjectsManager::ObjectsManager(TaskConfig& taskConfig) : mTaskName(taskConfig.taskName)
 {
   startPublishing(&mObjectsList, MonitorObject::SYSTEM_OBJECT_PUBLICATION_LIST);
+  mMonitorObjects.SetOwner(true);
 }
 
 ObjectsManager::~ObjectsManager()
 {
-  for (auto& mMonitorObject : mMonitorObjects) {
```

Comment:
No deletion of the MonitorObjects any more ?

Reply:
`TObjArray mMonitorObjects` is the explicit owner of MOs now (a few lines above).

Reply:
I saw it but did not realize that the deletion of a TObjArray destroys the pointers it contains.

Reply:
I think it does - [docu](https://root.cern.ch/doc/master/classTObjArray.html#a30ea50379bcd56377448529f781dfb6a)

TObjArray::~TObjArray()
Delete an array.
Objects are not deleted unless the TObjArray is the owner (set via SetOwner()).

Reply:
You are correct.